### DISCUSSION TITLE
How to regrid pft-level variables on a ne30pg3 grid?

### DISCUSSION THREAD

**Discussion by olyson** [ESCOMP/CTSM#2082] [General]
Does anyone have a workflow for regridding pft-level variables on a ne30pg3 grid to a, e.g, 1deg lat/lon grid?  We usually use pfts1d_ixy and pfts1d_jxy to map "p" to a 2d lat/lon grid.  However, in the case of ne30pg3 output, those variables are not actually 2d, i.e., pfts1d_ixy=48600 and pfts1d_jxy=1.

**Comment by wwieder** [comment]
I have something that works for pft level output on the "regular" SE grid.  We could try and adapt it to the ne30.  I'll post my script later today.

**Reply by wwieder** [reply]
Here's a script that I borrowed heavily from a ESDS post Deepak made
https://github.com/wwieder/ctsm_py/blob/master/notebooks/PFT_H1_gridded.ipynb

A local copy is here
/glade/u/home/wwieder/python/ctsm_py/notebooks/PFT_H1_gridded.ipynb

**Reply by wwieder** [reply]
There may be other post processing tools we need to develop for using the new ne30 grid?

This could be a good time to revisit our contributions to the ctsm-py repo?

**Comment by olyson** [comment]
That script you provided is for f09 history files though.
